src/Bridge/Faker/Provider/EnumProvider.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the "elao/enum" package.
+ *
+ * Copyright (C) 2017 Elao
+ *
+ * @author Elao <contact@elao.com>
+ */
+
+namespace Elao\Enum\Bridge\Faker\Provider;
+
+use Elao\Enum\EnumInterface;
+
+class EnumProvider
+{
+    /**
+     * Enum mapping as an array with :
+     *
+     * - alias for Enum class as key
+     * - Enum FQCN as value
+     *
+     * Example :
+     *
+     * ```
+     * [
+     *   'Civility' => Civility::class,
+     *   'Gender'   => Gender::class,
+     * ]
+     * ```
+     *
+     * @var array $enumMapping
+     */
+    private $enumMapping = [];
+
+    public function __construct(array $enumMapping)
+    {
+        foreach ($enumMapping as $enumAlias => $enumClass) {
+            $this->ensureEnumClass($enumClass);
+            $this->enumMapping[$enumAlias] = $enumClass;
+        }
+    }
+
+    /**
+     * @param string $enumValueShortcut As <ENUM_CLASS_ALIAS::ENUM_VALUE_CONSTANT>
```

Comment:
Eh eh, `<` and `>` is just a convention for a template mask but as a matter of fact, it can be ambiguous in an Alice/Fixture context.